pkg/controller/instance/controller_reconcile.go

```diff
@@ -273,6 +273,8 @@ func (igr *instanceGraphReconciler) updateResource(
 	// TODO: Handle annotations
 	desired.SetResourceVersion(observed.GetResourceVersion())
 	desired.SetFinalizers(observed.GetFinalizers())
+	desired.SetOwnerReferences(observed.GetOwnerReferences())
```

Comment:
Is this for adoption use cases ? 
What about setting ownerref to the instance ?